src/atomate2/vasp/jobs/pheasy.py

```diff
@@ -0,0 +1,61 @@
+"""Define the PhononDisplacementMaker for VASP."""
+
+from dataclasses import dataclass, field
+
+from atomate2.vasp.jobs.base import BaseVaspMaker
+from atomate2.vasp.sets.base import VaspInputGenerator
+from atomate2.vasp.sets.core import StaticSetGenerator
+
+
+@dataclass
+class PhononDisplacementMaker(BaseVaspMaker):
+    """
+    Maker to perform a static calculation as a part of the finite displacement method.
+
+    The input set is for a static run with tighter convergence parameters.
+    Both the k-point mesh density and convergence parameters
+    are stricter than a normal relaxation.
```

Comment:
is this the same as in jobs/phonons.py? if so, simply import :)

Reply:
Did it and delete the vasp/jobs/pheasy.py instead of directly importing from vasp/jobs/phonons.